dmoj/packet.py

```diff
@@ -152,7 +157,7 @@ def _read_forever(self):
         except KeyboardInterrupt:
             pass
         except Exception:  # connection reset by peer
-            traceback.print_exc()
+            log.exception('Exception while reading packet from site, will not attempt to reconnect! Quitting judge.')
```

Comment:
I don't see how this is useful. `_read_single` will swallow all socket errors by attempting to reconnect.